src/SixLabors.Core/MathF.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.Runtime.CompilerServices;
 
-#if NETCOREAPP2_0
+#if NETCOREAPP2_0 || NETCOREAPP2_1
```

Comment:
I wonder if there's a smarter check we can do to avoid having `|| NETCOREAPP2_N+1`?

Reply:
I use this in inside the csproj for Magick.NET:
```xml
  <PropertyGroup Condition="$(TargetFramework.StartsWith('netstandard'))">
    <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
  </PropertyGroup>
```

I could create a PR for this gets merged or @iamcarbon could add it to the current PR if he prefers that.

Reply:
Yeah, thought about this. Good to set the prescient now. Updated the PR to define a symbol.